chanzuckerberg/single-cell-data-portal
# feat: improved rollup pipeline changes

## Reason for Change

- #6399 
- If the reason for this PR's code changes are not clear in the issue, state value/impact

## Changes

- added a new pipeline step to generate a cell type ancestors fixture that will be used to perform the explode operation in the backend

## Testing steps

- Added unit tests for the ancestors fixture.
- Test run of the pipeline is currently being run in rdev. This message will be updated upon completion to indicate success or failure.
  - See run here: https://us-west-2.console.aws.amazon.com/batch/home?region=us-west-2#jobs/ec2/detail/db36f520-37b1-42e5-922c-97de42b0b9e1
